css/main.css

```diff
@@ -38,1357 +40,94 @@ ul {
-
-    font-family: 'Circe Regular';
-}
-
-.individual__price {
-    margin-top: 20px;
-    margin-bottom: 25px;
-    text-align: center;
-    font-weight: 700;
-    font-size: 18px;
-    font-family: 'Playfair Display', serif;
-}
-
-.individual__button {
-    margin-top: 20px;
-}
-
-.circle-img {
-    margin-top: 20px;
-}
-
-.circle-card {
-    position: relative;
-}
-
-.circle-card::after {
-    position: absolute;
-    top: 50%;
-    left: 50%;
-    display: block;
-    width: 19px;
-    color: #000;
-    content: '1кг';
-    font-weight: 700;
-    font-size: 14px;
-    transform: translateX(-51%) translateY(-50%);
-}
-
-.individual__name-circle {
-    margin-bottom: 45px;
-    text-align: center;
-    font-size: 14px;
-    font-family: 'Circe Regular';
-}
-
-.desing-img {
-    position: relative;
-}
-
-.desing-img::after {
-    position: absolute;
-    top: 180px;
-    left: 180px;
-    display: block;
-    color: #838383;
-    content: '1/35';
-    font-size: 12px;
-}
-
-.individual__name-text--design {
-    color: #2f2f2f;
-    text-align: center;
-    font-weight: 700;
-    font-size: 14px;
-}
-.individual__form{
-    background: #fff;
-    padding-bottom: 20px;
-}
-.individual__form-container{
-    display: flex;
-    flex-direction: column;
-    align-items: center;
-}
-.individual__form-title {
-    font-weight: 700;
-
-    font-family: 'Playfair Display', serif;
-}
-.individual__card--last{
-    margin-bottom: 0;
-}
-
-.individual__title--form {
-    margin: 0;
-    padding-top: 30px;
-    text-align: left;
-}
-
-.individual__form-result {
-    display: flex;
-    align-items: center;
-    flex-direction: column;
-}
-
-.individual__form-title {
-    color: #f50049;
-    font-size: 20px;
-    font-family: 'Intro Script R H2 Base';
-}
-
-.individual__form-text {
-    margin: 8px 0 10px 0;
-    padding-top: 5px;
-    width: 243px;
-    height: 30px;
-    border: 1px solid #dedede;
     border-radius: 50px;
-    text-align: center;
-    font-family: 'Circe Regular';
-}
-
-.form-button {
-    margin-top: 20px;
-}
-
-.about-us {
-    background-color: #fbebea;
-}
-
-.about-us__title {
-    margin: 0;
-    padding-bottom: 36px;
-    text-align: center;
-    font-size: 24px;
-    font-family: 'Playfair Display', serif;
-}
-
-.about-us__feedback {
-    display: flex;
-    align-items: center;
-    flex-direction: column;
-
-    font-family: 'Circe Regular';
-}
-
-.feedback__text {
-    margin-top: 7px;
-    width: 230px;
-    text-align: center;
-
-    font-size: 12px;
-}
-
-.feedback__title {
-    margin-top: 4px;
-    font-weight: 700;
-    font-size: 14px;
-}
-
-.feedback__link {
-    position: relative;
-    color: #f87b91;
-    text-decoration: underline;
-    font-style: italic;
-}
-
-.feedback__link::after {
-    position: absolute;
-    left: 85px;
-    content: '1/26';
+    background-color: #f50049;
 
-    font-style: normal;
+    font-size: 13px;
 }
 
-.feedback__link:hover {
-    cursor: pointer;
+.button_small {
```

Comment:
As far as `small` is modifier, rename it to `button--small`

Reply:
[x]